templates/cloud_sidebar.hbs

```diff
@@ -58,6 +58,8 @@
                     <li><a {{#if (active_content request.spin-path-info "/cloud/data-postgres" )}} class="active"
                             {{/if}} href="{{site.info.base_url}}/cloud/data-postgres">Persisting Data With
                             PostgreSQL</a></li>
+                    <li><a {{#if (active_content request.spin-path-info "/cloud/noops-sql-db" )}} class="active"
```

Comment:
This isn't really a tutorial... I think this needs to move to replace lines 114-115 (once the folder IDs are all cleared up)